src/openssl/algorithm.cr

```diff
@@ -0,0 +1,28 @@
+require "openssl"
+
+module OpenSSL
+  enum Algorithm
```

Comment:
Could you please add a short documentation to this enum type and the method?

Reply:
Well tried hard to add something meaningful (but I'm not sure it helps).
I'm really open to suggestions.

Reply:
I'm still unconfortable with the phrasing.
I dont even know if it is appropriate to cite `EVP_MD` and the acronym **EVP**.

Reply:
I've changed the phrasing trying to be clearer and at the same time not mention technical terms of the *LibCrypto API*.